README.md

```diff
@@ -110,7 +110,7 @@ For Linux:
 
 ```
 docker pull chef/inspec
-function inspec { docker run -it --rm -v $(pwd):/share chef/inspec "$@"; }
+function inspec { docker run -it --rm -v $(pwd):/share -v /var/run/docker.sock:/var/run/docker.sock chef/inspec "$@"; }
```

Comment:
The -v needs an explanation that it is there to allow scanning containers on the same host. You could add a separate command alias for that.